src/app/components/RemainingTime.js

```diff
@@ -22,6 +25,8 @@ function handleTimeChange(element) {
     }
 
     element.textContent = `Time Left: ${minToDisplay}m ${secToDisplay}s`;
```

Comment:
You change the value passed as an argument. Could there be another way to do it?

Reply:
An argument is a div, which contains changing text. We can put there <p> or <span> with text content, but does it change anything?

Reply:
I declared a variable `const parameter = element`, and used this here. Is it good practice?